02-setup.md

```diff
@@ -9,38 +9,6 @@ minutes: 10
 > *   Explain which initialization and configuration steps are required once per computer,
```

Comment:
Why are we saying "configuration per computer", and "per repository". We only explain here the steps needed to configure Git "per computer". If this is refer to the `--global` file, I think the learning objectives can be simplified to:

- Configure `git` the first time is used on a computer
- Understand the meaning of the `--global` configuration flag